notebook/static/notebook/js/outputarea.js

```diff
@@ -942,7 +942,7 @@ define([
     };
 
 
-    OutputArea.prototype.clear_output = function(wait, ignore_que) {
+    OutputArea.prototype.clear_output = function(wait, ignore_clear_queue) {
```

Comment:
It took me a while of staring at it to figure out the double negative logic and what it was supposed to say :).